### PR TITLE
Feat/realtime landing page

### PR DESCRIPTION
**Pull request by MildTomato**
Screenshot 2022-08-18 at 9 33 57 AM
Screenshot 2022-08-18 at 9 34 07 AM

### CODE REVIEW

File: apps/www/data/Solutions.json [hunk 1]
```diff
@@ -30,5 +30,13 @@
     "description_short": "",
     "label": "",
     "url": "/edge-functions"
+  },
+  "realtime": {
+    "name": "Realtime",
+    "icon": "M13 10V3L4 14h7v7l9-11h-7z",
+    "description": "Listen to changes in a Postgres Database and broadcast them over web sockets",
```
Comment (kiwicopple) [inline apps/www/data/Solutions.json:37]: perhaps this should be a bit broader now?

File: apps/www/pages/realtime/Realtime.tsx [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+        description={meta_description}
+        openGraph={{
+          title: meta_title,
+          description: meta_description,
+          url: `https://supabase.com/realtime`,
+          images: [
+            {
+              // [TODO] Update OG image
+              // [TODO] Update to use an appropriate icon for realtime in Solutions.json
+              url: `https://supabase.com${basePath}/images/product/auth/auth-og.jpg`,
```
Comment (MildTomato) [inline apps/www/pages/realtime/Realtime.tsx:61]: @MildTomato replace

**Review by MildTomato — COMMENTED** [review]
.

File: apps/www/data/Solutions.json (hunk 1 above)
Comment (MildTomato) [reply apps/www/data/Solutions.json]: "Create multiplayer experiences by sharing, broadcasting, and listenning to changes from other clients or the Database"

Comment (MildTomato) [reply apps/www/data/Solutions.json]: used Chases suggestions

**Review by joshenlim — APPROVED** [review]

**Pull request merged**